docs/docs/configuration.md

```diff
@@ -204,6 +204,24 @@ For example:
 # In this example, the following will not be packaged in to MTA archive:
 # all files with the"txt" extension and the "mtaignore" folder within the "myfolder" directory. 
      
+```
+From MBT v1.2.25 version, build-parameters ignore attribute will be upgraded to support [Full Glob Pattern](https://en.wikipedia.org/wiki/Glob_(programming)).
```

Comment:
From the MBT 1.2.25 version, the 'build-parameters ignore' attribute will be upgraded to support [Glob Patterns](https://en.wikipedia.org/wiki/Glob_(programming)).

Reply:
I prefer do not delete "Full", because currently mbt is partially support glob pattern